File: 05_hash_tables/python/02_check.py
'''
@Description: In User Settings Edit
@Author: your name
@Date: 2019-08-23 20:38:59
@LastEditTime: 2019-08-23 20:42:03
@LastEditors: Please set LastEditors
'''
obj = {}

def check(name):
    if(obj.get(name)):
        print("Hello World")
    else:
        obj[name] = True
        print("Hello World22")

check('a')
check('c')
check('a')

File: 05_hash_tables/es6/02_check.js
/*
 * @Description: In User Settings Edit
 * @Author: your name
 * @Date: 2019-08-23 20:35:16
 * @LastEditTime: 2019-08-23 20:36:49
 * @LastEditors: Please set LastEditors
 */
let obj ={}

const check = name => {
    if(obj[name]){
        console.log('存在')
    }else{
        obj[name] = true
        console.log('不存在')
    }
}

check('11')
check('22')
check('11')